### PR TITLE
Fix bug with selecting first input within ul if there button as main element

### PR DESCRIPTION
**Pull request by WorstOfAny**

### CODE REVIEW

File: cccombo.js [hunk 1]
```diff
@@ -189,7 +189,8 @@ CccomboList.prototype = {
 function Cccombo(element) {
 	// Constructor
 	this.element = element;
-	this.input = this.element.querySelector('input:not([type="hidden"])');
+	this.input =
+		this.element.querySelector('.cccombo > input:not([type="hidden"])');
```
Comment (AlexWayfer) [inline cccombo.js:193]: Let's use [`:scope`](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope), [it supported by IE](https://github.com/mdn/browser-compat-data/issues/846).

**Review by AlexWayfer — CHANGES_REQUESTED** [review]

File: cccombo.js (hunk 1 above)
Comment (WorstOfAny) [reply cccombo.js:193]: Done.

Comment (AlexWayfer) [reply cccombo.js:193]: Sorry, my mistake, let's use `.cccombo >`. `:scope` isn't supported by IE.

Comment (WorstOfAny) [reply cccombo.js:193]: back done

**Review by AlexWayfer — APPROVED** [review]

**Pull request merged**